.gitignore

```diff
@@ -7,7 +7,7 @@
 /docs/test-coverage
 
 # dependencies
-/node_modules
+node_modules/
```

Comment:
We now use Yarn workspaces as the `docs` project uses a version of TypeScript different from the other projects in our Nx workspace. Because of this, we have more than one `node_modules` directory.